docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3118,9 +3119,33 @@ error rate is not very intuitive for the end user.
 At some point in the future, we might make this more user friendly. However, for the 
 sake of a minimum functional model, this is satisfactory. 
 
-One function has an additional parameter. The typographic function needs an input for 
-the probability that a corrupted token contains the original token. For simplicity, 
-we will use 0.1 and the user will not be able to edit this value. 
+A few functions have additional parameters that allow the user more control. These 
+inputs will not be specified in the main settings for pseudopeople listed above. We 
+imagine that a yaml file (or similar file type) will be available for users to download 
+and adjust the additional inputs if desired. The table below outlines these inputs, 
+their purpose, and the default value for the yaml file. 
+
+.. list-table:: Additional Inputs and Default Values
+  :widths: 20 20 20 20
+  :header-rows: 0
+
+  * - Noise Function Affected
+    - Observer Data Affected 
+    - Additional Inputs 
+    - Default Input Value 
+  * - Typographic Noise
+    - First name, middle initial, last name, street name, city name, employer name 
+    - Probability that a corrupted token contains the original token
+    - 0.1 
+  * - Numeric miswriting 
```

Comment:
This implies that numeric miswriting works significantly differently between age, ZIP, and other numbers. Why not just separate these into different noise functions? IIUC, that would make the second column of this table unnecessary.

Reply:
Yeah, I can split this into multiple functions. I think the second column is still helpful for clarity even if I agree its technically redundant now.